pkg/sql/distsql_running.go

```diff
@@ -261,10 +261,10 @@ func (dsp *DistSQLPlanner) setupFlows(
 	defer physicalplan.ReleaseSetupFlowRequest(&localReq)
 	ctx, flow, err := dsp.distSQLSrv.SetupLocalSyncFlow(ctx, evalCtx.Mon, &localReq, recv, localState)
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, false, err
 	}
 
-	return ctx, flow, nil
+	return ctx, flow, setupReq.EvalContext.Vectorize != int32(sessiondata.VectorizeOff), nil
```

Comment:
why don't we check here if the flow was vectorized or not? Why do we look at a setting